WordPress/Classes/ViewRelated/Blog/NoResultsViewController.swift

```diff
@@ -101,6 +101,13 @@ import WordPressAuthenticator
         accessorySubview = accessoryView
     }
 
+    /// Public method to remove No Results View from parent view.
+    ///
+    @objc func removeFromView() {
+        view.removeFromSuperview()
```

Comment:
I think Apple recommends calling `willMove(toParentViewController: nil)` before performing these other two steps. It looks like they've taken the documentation page I used to use for reference, but we use these same steps in some other places in the app.

Reply:
Thanks for the tip @frosty!